Assets/PatchKit Patcher/Editor/PatcherManifestCreator.cs

```diff
@@ -10,30 +10,29 @@ public static class PatcherManifestCreator
     {
         private const int ManifestVersion = 4;
 
-        [MenuItem("Tools/PatchKit Patcher/Manifest/Windows")]
-        private static void Test(Manifest manifest)
+        private static void SaveTestManifest(Manifest manifest)
         {
             string targetLocation = EditorUtility.SaveFilePanel("Choose test manifest location", "", "patcher.manifest", "test");
 
             File.WriteAllText(targetLocation, JsonConvert.SerializeObject(manifest, Formatting.Indented));
         }
 
         [MenuItem("Tools/PatchKit Patcher/Manifest/Windows")]
```

Comment:
If we are not removing these methods, can we wrap them inside `#if` with some new define named `PATCHKIT_PATCHER_UNITY_DEV` or use another if such already exist?